package com.ramy.bugreport.repository;

import java.util.UUID;

import org.springframework.data.jpa.repository.JpaRepository;

import com.ramy.bugreport.domain.Component;

public interface IComponentRepository extends JpaRepository<Component, UUID> {
}
